doc/source/user_guide/text.rst

```diff
@@ -262,25 +262,30 @@ i.e., from the end of the string to the beginning of the string:
                    '', np.nan, 'CABA', 'dog', 'cat'],
```

Comment:
The sentence just before this example ("replace by default replaces regular expressions") can probably also be updated?